source/NVDAObjects/window/_msOfficeChart.py

```diff
@@ -561,8 +561,12 @@ def _get_description(self):
 				# Translators: Indicates that there is 1 series in a chart.
 				seriesValueString = _( "There is 1 series in this chart" )
 			else:
```

Comment:
I think you can eliminate the above condition right? as the new code exists to cover singular vs plural

Reply:
Note that in some languages, e.g. some Slavic ones, it seems that they use the singular after 21, 31, 101, 1001, etc., at least when we read the formula in the .po file.

So we cannot simplify the code keeping the same sentence, because in these languages, we would have "total" for 5, 6, 7, 15 or 40, but not for 1, 21, 31 or 41.

Of course, it's quite a theoretical case since the inconsistency would occur for a graphic with at least 21 series, which is unlikely.
But to cover this case anyway, I have removed the word "total" which is not mandatory at all in any case.

Cc @zstanecic to check that I do not write wrong assumptions on Slavic languages that I do not speak personally.